src/ec/packages/ec-component-footer-standardised/footer-standardised.html.twig

```diff
@@ -0,0 +1,99 @@
+    - "title" (optional) (string) OR (object with Link component in property)
+    - "title_class_name" (optional) (string) (default: '')
+    - "description" (optional) (string) (default: '')
+    - "content_before" (optional) (string) (default: '')
+    - "list_class_name" (optional) (string) (default: '')
+    - "links" (optional) (array of Link components) (default: []):
+  - "extra_classes" (optional) (string) (default: '') Extra classes (space separated) for the icon
+  - "extra_attributes" (optional) (array) (default: []) Extra attributes for icon
+    - "name" (string) Attribute name, eg. 'data-test'
+    - "value" (string) Attribute value, eg: 'data-test-1'     
+#}
+
+{# Internal properties #}
+{% set _sections = sections|default([]) %}
+
+{% set _css_class = '' %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% if attr.value is defined %}
+      {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value|e('html_attr') ~ '"' %}
+    {% else %}
+      {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name %}
+    {% endif %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+
+<footer class="ecl-footer-standardised"{{ _extra_attributes|raw }}>
+  {# Main container #}
+  <div class="ecl-container ecl-footer-standardised__container">
+    {% for section in _sections %}
+      {# Section #}
+      <div class="ecl-footer-standardised__section {{ 'ecl-footer-standardised__section' ~ loop.index }}">
+        {# content before #}
+        {% if section.content_before is defined and section.content_before is not empty %}
+         <div class="ecl-footer-standardised__content">{{ section.content_before|e }}</div>
+        {% endif %}
+        {# title #}
+        {% if section.title is defined and section.title is not empty %}
+          {% if section.title.link is defined and section.title.link is not empty %}
+            {% include '@ecl-twig/ec-component-link/link.html.twig' with section|merge({
+              link: section.title.link|merge({ type: 'standalone'}),
+              extra_classes: 'ecl-footer-standardised__title',
+              icon: item.icon|default({})
+            }) only %}
+          {% else %}
+            <div class="ecl-footer-standardised__title
+            {{- section.title_class_name is defined and section.title_class_name is not empty ? ' ' + section.title_class_name -}}">
+            {{- section.title -}}</div>
+        {% endif %}
+      {% endif %}
+      {# description #}
+      {% if section.description is defined and section.description is not empty %}
+        <div class="ecl-footer-standardised__description">{{ section.description|e }}</div>
```

Comment:
Also here you set the escape but being a description this text might contain many characters that would be potentially escaped, we normally treat this with a |raw "filter" which kind of does the opposite. If you are doing this because you think it's the right thing to do let's talk about it, anyway.. ;)

Reply:
yep see above